src/SDKs/CustomProviders/Microsoft.CustomProviders/Microsoft.CustomProviders.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
```

Comment:
Please rename this directory to `Management.CustomProviders` and rename this file to `Microsoft.Azure.Management.CustomProviders.csproj`